docs/SAS_URL.md

```diff
@@ -0,0 +1,5 @@
+# How to generate a SAS URL to access an Azure table
+
+Once the subscription table created in the storage account, go on the storage account panel on [portal.azure.com](https://portal.azure.com) and click on `Sharedaccess signature`. Fill the form as follow and click `Generate SAS and connection string`:
+
```

Comment:
Add an instruction that they need to provide us with the Table service SAS URL.

Reply:
I already addressed this comment, see line 7